conformance/02_push_test.go

```diff
@@ -220,7 +219,10 @@ var test02Push = func() {
 				Expect(err).To(BeNil())
 				Expect(resp.StatusCode()).To(Equal(http.StatusNoContent))
 				Expect(resp.Header().Get("Location")).ToNot(BeEmpty())
-				Expect(resp.Header().Get("Range")).To(Equal(fmt.Sprintf("bytes=%s", testBlobBChunk1Range)))
+				Expect(resp.Header().Get("Range")).To(SatisfyAny(
+					Equal(testBlobBChunk1Range), // Allow missing "bytes=" prefix
+					Equal(fmt.Sprintf("bytes=%s", testBlobBChunk1Range)),
+				))
```

Comment:
added this change to allow for missing "bytes=" prefix

Reply:
chatting with @rchincha - looks like this is required: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range

working on zot version with this fix

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range

The <unit> doesn't look optional, so this appears to be a zot bug and addressed in https://github.com/project-zot/zot/pull/1341

Reply:
To follow the robustness principle, I believe we need to be strict in what registries output here, only allowing the output without `bytes=`. Registries and clients can be liberal in what they accept from each other, but for conformance, their output should follow the distribution-spec.